### PR TITLE
Declare encoding for emails in their headers.

### LINKED ISSUE #94: Move the Twisted application(s) to a .tac file
Right now Twisted is being used by calling ``reactor.run()`` in ``fmn.consumer.backend``[0]. The more common approach is to define an application in a .tac file. This is a good place to configure everything required for the application to run (logging comes to mind).

The application can then be run with ``twistd -y path/to/tac`` which starts the reactor.

See https://twistedmatrix.com/documents/current/core/howto/application.html for more information.

[0] https://github.com/jeremycline/fmn.consumer/blob/3009eee0a2cc16d528ecf34ed2a9ddf39aa41cc8/fmn/consumer/backend.py#L231

### PR DESCRIPTION
**Pull request by ralphbean**
This might fix fedora-infra/fmn#94.  ;)

### CODE REVIEW

**Comment by pypingou** [comment]
:+1: for me

**Comment by ralphbean** [comment]
Thanks @pypingou.  I'll wait until we hear back from @Rombobeorn in fedora-infra/fmn#94 (or a few days.. whichever comes first).

File: fmn/consumer/backends/mail.py [hunk 1]
```diff
@@ -79,7 +79,12 @@ def send_mail(self, session, recipient, subject, content,
         if footer:
             content += u'\n\n--\n{0}'.format(footer.strip())
 
-        email_message.set_payload(to_bytes(content))
+        email_message.set_payload(to_bytes(content), 'utf-8')
+
+        # Explicitly declare encoding, but remove the transfer encoding
+        # https://github.com/fedora-infra/fmn/issues/94
+        email_message.set_charset('utf-8')
+        del msg['Content-Transfer-Encoding']
```
Comment (rtnpro) [inline fmn/consumer/backends/mail.py]: Shouldn't `msg` be `email_message`?

Comment (ralphbean) [inline fmn/consumer/backends/mail.py]: Definitely!

**Pull request merged**